homeassistant/components/media_player/sonos.py

```diff
@@ -874,6 +887,15 @@ def set_volume_level(self, volume):
         """Set volume level, range 0..1."""
         self._player.volume = str(int(volume * 100))
 
+    @soco_error
+    def set_shuffle(self, shuffle):
+        """Enable/Disable shuffle mode."""
+        self._shuffle = shuffle
+        if shuffle is True:
```

Comment:
**Preference:**
Instead of this block, you can just do:
```python
self._player.play_mode = 'SHUFFLE' if shuffle else 'NORMAL'
```

Reply:
This is now addressed. Thanks for the suggestion